eth/db/account.py

```diff
@@ -49,7 +49,7 @@
     validate_canonical_address,
 )
 from eth.tools.logging import (
-    TraceLogger
+    EthereumLogger
```

Comment:
I think I'm even less sold on this naming ;) Basically because there's nothing Ethereum specific with this logger. So I'd rather name this `[Custom|Extended|FancyPants]DebugLogger` or something like that.

Reply:
@pipermerriam - is CustomDebugLogger looking ok?

Reply:
I think I like just plain `DebugLogger` or maybe `ExtendedDebugLogger`.  "Custom" is a bit too generic.